hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -90,6 +90,11 @@
   // It is here so that both the client and deltastreamer use the same reference
   public static final String DELTASTREAMER_CHECKPOINT_KEY = "deltastreamer.checkpoint.key";
 
+  public static final ConfigProperty<String> DATABASE_NAME = ConfigProperty
+      .key("hoodie.database.name")
+      .defaultValue("default")
```

Comment:
Do not set the default value to keep compatibility.

Reply:
If there is no default value, is it must be set? like tableName

Reply:
Would it be better if databaseName had a default value?like hive

Reply:
It's not like hive. Users can write by Spark dataframe. That set `default` as the default value will mislead hudi to do something wrong.